src/pa01/snakes_and_ladders.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Kristian Gunder Kramås, Anders Mathiesen'
+__email__ = 'kristiakr@nmbu.no, andermat@nmbu.no'
+
+
+def single_game(num_players):
+    """
+        Returns duration of single game.
+
+        Arguments
+        ---------
+        num_players : int
+            Number of players in the game
+
+        Returns
+        -------
+        num_moves : int
+            Number of moves the winning player needed to reach the goal
+        """
+    import random
+    ladders = {1: 39, 8: 2, 36: 16, 43: 19, 49: 30, 65: 17, 68: 14}
+    snakes = {24: -19, 33: -30, 42: -12, 56: -19, 64: -37, 74: -62, 87: -17}
```

Comment:
Cool approach using the difference instead of absolute position. I think this is how one would start approaching the problem if one wanted to solve this using linear algebra and get a probability distribution (i.e. continuous histogram) out.